project_wbs/models/project_project.py

```diff
@@ -0,0 +1,342 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Luxim d.o.o.
+# Copyright 2017 Matmoz d.o.o.
+# Copyright 2017 Deneroteam.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import time
+from datetime import datetime, date
+from openerp import api, fields, models, _
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+    _description = "WBS element"
+    _order = "c_wbs_code"
+
+    @api.multi
+    def _get_project_analytic_wbs(self):
```

Comment:
c_wbs_code is a composed code that reflects the position of the project inside the work breakdown hierarchy and it is used mostly to sort the project list regarding the wbs hierarchy. The project list's sorting order would be random hierarchy wise without it since the hierarchy is not static as with the accounts, but dynamic - a project can be moved to another parent in time and that way moved to another spot in the hierarchy order.